### PR TITLE
#12266: Cleanup ternary backward

### PR DESCRIPTION
**Pull request by Aswinmcw**
Tracking Issue : #12266 

### Checklist
- [x] Post commit CI passes - https://github.com/tenstorrent/tt-metal/actions/runs/10735726353
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by VirdhatchaniKN — APPROVED** [review]
LGTM

File: ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward.hpp [hunk 1]
```diff
@@ -122,19 +66,16 @@ struct ExecuteTernaryBackwardLerp {
 }  // operations::ternary_backward
 
 //type 1
-constexpr auto addcmul_bw = ttnn::register_operation_with_auto_launch_op<
+constexpr auto addcmul_bw = ttnn::register_operation<
```
Comment (patrickroberts) [inline ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward.hpp:69]: Can you show me where the test is for this operation? I'd just like to make sure that removing the auto launch is indeed the correct thing to do here.

Comment (Aswinmcw) [reply ttnn/cpp/ttnn/operations/eltwise/ternary_backward/ternary_backward.hpp:69]: @patrickroberts the test is located at `tests/ttnn/unit_tests/operations/backward/test_backward_addcmul.py`
I have verified the tests and they are passing

**Review by eyonland — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
Available in https://github.com/tenstorrent/tt-metal/pull/12691 - Hence closing PR

**Pull request closed without merging**